whylog_vim/proxy/__init__.py

```diff
@@ -0,0 +1,66 @@
+from whylog import whylog_factory
+from whylog_vim.proxy.teacher import TeacherProxy
+from whylog_vim.proxy.log_reader import LogReaderProxy
+from whylog_vim.consts import MainStates as States, ActionTypes
+from whylog_vim.exceptions import UnknownAction
+
+
+class WhylogProxy():
+
+
+    def __init__(self, editor):
+        self.editor = editor
+        log_reader, teacher_generator = whylog_factory()
+        self.teacher_generator = teacher_generator
+        self.log_reader = LogReaderProxy(editor, log_reader)
+        self.new_teacher()
```

Comment:
Oh, really? I thought that this is e field new_teacher and mistake here. Maybe rename, add a verb to method name?